pkg/mkimage-raw-efi/install

```diff
@@ -170,24 +171,82 @@ done
 # we may be asked to pause before install procedure
 grep -q eve_pause_before_install /proc/cmdline && pause "formatting the /dev/$INSTALL_DEV"
 
-# FIXME: gross hack to allow P3 partition on a disk different from a bootable one
-# this needs to go away the moment we implement LVM or ZFS
+P3_ON_BOOT_PLACEHOLDER="P3_ON_BOOT_PLACEHOLDER"
+POOL_CREATION_COMMAND=""
+DISK_WITH_P3=""
+DISKS_TO_MERGE_COUNT=0
 if [ "$INSTALL_DEV" != "$INSTALL_PERSIST" ]; then
-   # apparently sgdisk -Z doesn't clear MBR and keeps complaining
-   PDEV="/dev/$INSTALL_PERSIST"
-   dd if=/dev/zero of="$PDEV" bs=512 count=1 conv=notrunc
-   sgdisk -Z --clear "$PDEV" 2>/dev/null || :
-   sgdisk --new 1:2048:0 --typecode=1:5f24425a-2dfa-11e8-a270-7b663faccc2c --change-name=1:P3 "$PDEV"
-   sgdisk -v "$PDEV"
-   # force make-raw to skip persist
-   MAKE_RAW_PARTS="efi imga imgb conf"
+   if echo "$INSTALL_PERSIST"| grep -q ","; then
```

Comment:
So what if I want zfs on a single P3 disk? I won't have a comma in my spec will I?

Reply:
I keep the old variant of deployment in this case, so, if EVE will see zfs-(kvm|xen|acrn) in the eve-release, storage-init will allocate zfs on the partition/disk.

Reply:
Can we somehow unify the single disk behaviour as well? I mean -- I'd be even fine with ```...=sda,``` so that the presence of the coma (even for a single disk) would trigger this

Reply:
Added support for this. You can just place comma to use partition on the same disk as an eve_install_disk.